routesrv/eskipbytes.go

```diff
@@ -50,6 +59,10 @@ func (e *eskipBytes) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	span := tracing.CreateSpan("serve_routes", r.Context(), e.tracer)
 	defer span.Finish()
 
+	if e.useEtag && e.hash != "" && r.Header.Get("Etag") == e.hash {
+		w.WriteHeader(http.StatusNotModified)
+		return
+	}
 	if data, initialized := e.bytes(); initialized {
 		w.Write(data)
```

Comment:
I think we'd need to set Etag header around here.
Also Etag header should follow certain format, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag